test/integration/015_cli_invocation_tests/test_cli_invocation.py

```diff
@@ -173,3 +234,60 @@ def _run_simple_dbt_commands(self, project_dir):
         # In case of 'dbt clean' also test that the clean-targets directories were deleted.
         for target in self.config.clean_targets:
             assert not os.path.isdir(target)
+
+
+class TestCLIInvocationWithProfilesAndProjectDir(ModelCopyingIntegrationTest):
+
+    @property
+    def schema(self):
+        return "test_cli_invocation_015"
+
+    @property
+    def models(self):
+        return "models"
+
+    @property
+    def custom_schema(self):
+        return "{}_custom".format(self.unique_schema())
+
+    def _test_postgres_sub_command_with_profiles_separate_from_project_dir(
+        self,
+        dbt_sub_command: str
+    ):
+        """
+        Test if a sub command runs well when a profiles dir is separate from a
+        project dir.
+        """
+        profiles_dir = "./tmp-profile"
+        workdir = os.getcwd()
+        with temporary_working_directory() as tmpdir:
+
+            profiles = get_custom_profiles_config(
+                self.database_host, self.custom_schema)
+            create_directory_with_custom_profiles(profiles_dir, profiles)
+
+            project_dir = os.path.relpath(workdir, tmpdir)
+            if os.path.exists(f"{project_dir}/profiles.yml"):
+                os.remove(f"{project_dir}/profiles.yml")
+
+            args = [
+                dbt_sub_command, "--profiles-dir", profiles_dir, "--project-dir", project_dir
+            ]
+            self.run_dbt(args, profiles_dir=False)
```

Comment:
I was able to get this test running locally by changing these lines to:
```suggestion
            project_dir = os.path.relpath(workdir, os.getcwd())
            if os.path.exists(f"{project_dir}/profiles.yml"):
                os.remove(f"{project_dir}/profiles.yml")

            other_args = [
                dbt_sub_command, "--profiles-dir", profiles_dir, "--project-dir", project_dir
            ]
            self.run_dbt(other_args, profiles_dir=False)
```
When I tried interactively executing, I found that:
- `args` is "special" and heavily used within the `test.integration.base` methods, so switching to a different variable name worked nicely
- The `project_dir` relative path didn't seem to be working right, I think because my working directory was not _actually_ `tmpdir`. Is the context manager silently failing when running locally? I'm not totally sure.

Here's the output from one of my previous failed runs:
```python
ipdb> workdir
'/private/var/folders/zf/01vkmdp51jj6hfmhfjglfhj00000gn/T/dbt-int-test-15liylvn'
ipdb> tmpdir
'/var/folders/zf/01vkmdp51jj6hfmhfjglfhj00000gn/T/tmp67cwv216'
ipdb> os.getcwd()
'/private/var/folders/zf/01vkmdp51jj6hfmhfjglfhj00000gn/T/tmp67cwv216'
ipdb> project_dir
'../../../../../../private/var/folders/zf/01vkmdp51jj6hfmhfjglfhj00000gn/T/dbt-int-test-15liylvn'
ipdb> os.listdir(project_dir)
*** FileNotFoundError: [Errno 2] No such file or directory: '../../../../../../private/var/folders/zf/01vkmdp51jj6hfmhfjglfhj00000gn/T/dbt-int-test-15liylvn'
```